admin/google_search_console/views/gwt-create-redirect.php

```diff
@@ -0,0 +1,61 @@
+<?php
+/**
+ * @package WPSEO\Admin|Google_Search_Console
+ *
+ * This is the view for the modal box that appears when the create redirect link is clicked
```

Comment:
Might be an idea to put all variables available for use in the comment so it is clearer and PHPStorm knows what's up. Like this:
```php
@var string $view_type The type of view to be displayed, can be 'create', 'already_exists', 'no_premium'
@var string $current_redirect The existing redirect
@var string $url Redirect for URL
```